api/client/options.proto

```diff
@@ -219,4 +220,7 @@ message CommandLineOptions {
   // "emit_previous_request_delta_in_response_header" to record elapsed time between request
   // arrivals.
   google.protobuf.StringValue latency_response_header_name = 36;
+  // Provide an execution starting date and time. Optional, any value specified must be in the
+  // future.
+  google.protobuf.Timestamp schedule = 105;
```

Comment:
I feel like schedule here implies the ability to create recurring requests.  How do you feel about scheduled_start?

Reply:
done here and all other applicable spots.